torch/csrc/inductor/aoti_runner/model_container_runner.h

```diff
@@ -35,6 +35,9 @@ class TORCH_API AOTIModelContainerRunner {
       const TensorConstantMap& const_map,
       bool use_inactive,
       bool validate_full_updates);
+  void run_const_fold(
+      bool use_inactive,
+      AOTInductorStreamHandle cuda_stream_handle = nullptr);
```

Comment:
You can add pybind for this in https://github.com/pytorch/pytorch/blob/main/torch/csrc/inductor/aoti_runner/pybind.cpp, and then you should be able to test in python.